third_party/amd/python/test/test_chained_dot_fp8.py

```diff
@@ -0,0 +1,164 @@
+"""
```

Comment:
I don't think we need to add a python test, since chained_matmul_fp8 is not a core functionality.
We can have a regression test in our perf-kernel dir, like the ones here: https://github.com/ROCm/triton/tree/main_perf/python/perf-kernels